src/server.ts

```diff
@@ -1144,7 +1144,8 @@ async function startServer() {
       try {
         const data = matchedData(req);
         const content = API.getBuildLog(data.root);
-        res.status(200).json({ content });
+        // eslint-disable-next-line no-control-regex
+        res.status(200).json({ content: content.replace(/\u001b/g, "\\u001b") });
```

Comment:
Maybe we can move the `.replace` call to `const content = API.getBuildLog(data.root).replace(/\u001b/g, "\\u001b");`